common/beerocks/bwl/CMakeLists.txt

```diff
@@ -91,19 +100,29 @@ elseif(BWL_TYPE STREQUAL "NL80211")
     )
 
     # Hostapd/NL80211 include directories
-    include_directories(
+    list(APPEND BWL_INC_DIRS
         ${HOSTAPD_DIR}/src/utils
         ${HOSTAPD_DIR}/src/common
         ${HOSTAPD_DIR}/src/drivers
-        ${PLATFORM_INCLUDE_DIR}/libnl-tiny
     )
+    # OpenWrt support only libnl-tiny
+    if (TARGET_PLATFORM STREQUAL "openwrt")
+        list(APPEND BWL_INC_DIRS ${PLATFORM_INCLUDE_DIR}/libnl-tiny)
```

Comment:
You can simply change this into
```
        include_directories(${PLATFORM_INCLUDE_DIR}/libnl-tiny)
```
without adding BWL_INC_DIRS variable.

However, Moran is making full nl3 a required dependency independent of the target (cfr. #1368) so you can simply remove this libnl-tiny option. In a separate commit, obviously.